src/plugins/auto/plugin.cpp

```diff
@@ -96,6 +96,31 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::ParseMetaDevices(cons
     // last token in the string (which has no comma after that)
     devicesWithRequests.push_back(priorities.substr(i, priorities.length() - i));
 
+    auto setTputAsDefault = [&](const std::string& targetDevice,
+                               std::map<std::string, std::string>& deviceConfig,
+                               const std::map<std::string, std::string>& mergedConfig) {
+        if (GetName() == "AUTO" && deviceConfig.find(PluginConfigParams::KEY_PERFORMANCE_HINT) == deviceConfig.end() &&
+            mergedConfig.find(targetDevice) == mergedConfig.end()) {
```

Comment:
when will mergedConfig be triggered here?

Reply:
you mean when below codes will be executed, right?
`mergedConfig.find(targetDevice) == mergedConfig.end()`

Reply:
yeah, what case it is targeting to handle?

Reply:
those codes will be triggered to check if any secondary properties setting to target device if all of below conditions are met.
1. plugin name is AUTO.
2. no performance hint setting to target device from user through AUTO plugin.

Reply:
can  you help paste some code snippets to explain your comment?

Reply:
MULTI set TPUT like AUTO if user doesn't specify performance hint to MULTI or to HW device. @wangleis Is it expected as we discuss?